emit-ch4plume-v1/README.md

```diff
@@ -1,5 +1,69 @@
+# Installation and Usage
+The steps below will walk you through setting up your own instance of the project. 
+
+## Install Project Dependencies
+To set up the development environment for this website, you'll need to install the following on your system:
+
+- [Node](http://nodejs.org/) v20 (To manage multiple node versions we recommend [nvm](https://github.com/creationix/nvm))
+- [npm](https://www.npmjs.com/) Package manager
+- [Yarn](https://yarnpkg.com/) Package manager
+
+## Install Application Dependencies
+
+If you use [`nvm`](https://github.com/creationix/nvm), activate the desired Node version:
+
+`nvm install`
+
+The desired node version is defined in `.nvmrc` file.
+To install and switch to the desired node version:
+```nvm use```
+
+Install Node modules:
+
+```yarn install```
+OR 
+```npm install```
+
+This command  will install all the dependencies and devtools needed for local development. 
+ 
+### Config files
+Configuration is done using [dot.env](https://parceljs.org/features/node-emulation/#.env-files) files.
+
+These files are used to simplify the configuration of the app and should not contain sensitive information.
+
+Copy the `.env.local-sample` to `.env` to add your configuration variables.
+
+`cp .env.local-sample .env`
+
+Replace `xxxxxxx` with keys and token.
+
+### Get Data
+
+To set the environment variables from the `.env` file, add export in front of each variable. 
+For example:
+
+`export MAP_STYLE=“xxxxxxxxxxxx”`
```

Comment:
Can you replace this by running 
`node --env-file .env update_data.js`
When running this command the developer should not have to export the env vars